target/linux/pistachio/creator-platform-default-cascoda.config

```diff
@@ -6,7 +6,7 @@ CONFIG_LIBCURL_OPENSSL=y
 CONFIG_OPENSSL_WITH_EC=y
 CONFIG_VERSIONOPT=y
 CONFIG_VERSION_NICK="Ci40"
-CONFIG_VERSION_REPO="http://downloads.creatordev.io/pistachio/marduk/packages"
+CONFIG_VERSION_REPO="http://downloads.creatordev.io/openwrt/ci40-v1.0.1/pistachio/marduk/packages"
```

Comment:
For the moment, yes, this is to just get the tip in sync before we get a proper solution.